### PR TITLE
archiver: Add documentation for "key export" and "key import" commands.

### PR DESCRIPTION
**Pull request by textshell**

### CODE REVIEW

File: borg/archiver.py [hunk 1]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
 
         key_parsers = subparser.add_subparsers(title='required arguments', metavar='<command>')
 
+        key_export_epilog = textwrap.dedent("""
+        If repository encryption is used, the repository is unaccessable
```
Comment (enkore) [inline borg/archiver.py]: inaccessible?

File: borg/archiver.py [hunk 2]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
...
+        without the key. This command allows to backup this essential key.
+
+        There are two backup formats. The normal backup format is suitable for
+        digital storage as a file. The ``--paper`` backup format is opimized for
```
Comment (enkore) [inline borg/archiver.py]: optimized

File: borg/archiver.py [hunk 3]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
...
+        print out and later type-in, with per line checks to reduce problems
```
Comment (enkore) [inline borg/archiver.py]: printing out and typing back in later (or so)?

File: borg/archiver.py [hunk 4]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
...
+        with manual input.
+
+        For repositories using keyfile encryption the key is saved locally
+        on the system that is capable of doing backups. To guard against loss
+        of this key the key needs to be backuped independent of the main data
```
Comment (enkore) [inline borg/archiver.py]: backed up

File: borg/archiver.py [hunk 5]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
...
+        backup.
+
+        For repositories using the repokey encryption the key is saved in the
+        repository in the config file. A backup is thus not strictly needed,
+        but guards against the repository becoming inaccessable if the file
```
Comment (enkore) [inline borg/archiver.py]: inaccessible

File: borg/archiver.py [hunk 6]
```diff
@@ -1134,9 +1153,17 @@ def build_parser(self, args=None, prog=None):
                                default=False,
                                help='Create an export suitable for printing and later type-in')
 
+        key_import_epilog = textwrap.dedent("""
+        This command allows to restore a key previously backuped with the
```
Comment (enkore) [inline borg/archiver.py]: backed up

File: borg/archiver.py (hunk 3 above)
Comment (textshell) [inline borg/archiver.py]: Not sure how to phrase this nicely. I‘ll think about improvements.

**Comment by textshell** [comment]
Thank, the spelling problems should be fixed now.

File: borg/archiver.py (hunk 1 above)
Comment (ThomasWaldmann) [inline borg/archiver.py]: [ui]naccessible

File: borg/archiver.py [hunk 7]
```diff
@@ -1120,9 +1120,28 @@ def build_parser(self, args=None, prog=None):
...
+        without the key. This command allows to backup this essential key.
+
+        There are two backup formats. The normal backup format is suitable for
+        digital storage as a file. The ``--paper`` backup format is optimized for
+        print out and later type-in, with per line checks to reduce problems
+        with manual input.
+
+        For repositories using keyfile encryption the key is saved locally
+        on the system that is capable of doing backups. To guard against loss
+        of this key the key needs to be backed up independent of the main data
```
Comment (ThomasWaldmann) [inline borg/archiver.py]: independently

File: docs/quickstart.rst [hunk 8]
```diff
@@ -228,6 +228,11 @@ For automated backups the passphrase can be specified using the
     the key in case it gets corrupted or lost. Also keep your passphrase
     at a safe place.
 
+    You can make backups using :ref:`borg_key_export` subcommand.
+
+    If you want to print a backup of your key to paper use the ``--paper``
+    option of this command and print the result.
+
     The backup that is encrypted with that key/passphrase won't help you
     with that, of course.
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst]: The "with that" here referred to "make a backup copy ... keep it at a safe place" some lines above.

Due to the new content you added in between, the reference is now somehow broken.

File: borg/archiver.py (hunk 1 above)
Comment (textshell) [inline borg/archiver.py]: i use inaccessible now. That seems a bit better.

File: borg/archiver.py (hunk 3 above)
Comment (textshell) [inline borg/archiver.py]: i‘ve rewritten that sentence.

File: docs/quickstart.rst (hunk 8 above)
Comment (textshell) [inline docs/quickstart.rst]: Yes, this is now more explicit i hope.

File: docs/quickstart.rst [hunk 9]
```diff
@@ -228,8 +228,13 @@ For automated backups the passphrase can be specified using the
...
 
-    The backup that is encrypted with that key/passphrase won't help you
-    with that, of course.
+    You can make backups using :ref:`borg_key_export` subcommand.
+
+    If you want to print a backup of your key to paper use the ``--paper``
+    option of this command and print the result.
+
+    A backup inside of the backup that is encrypted with that key/passphrase
+    won't help you with that, of course.
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst]: still "with that".

**Pull request merged**